modules/lightning_install/lightning_install.info.yml

```diff
@@ -5,3 +5,5 @@ hidden: true
 package: Lightning
 description: 'Provides helpful under-the-hood functionality used while installing Lightning.'
 version: '8.x-4.000-beta1'
+dependencies:
+  - drupal:system (>= 8.6)
```

Comment:
Why was this added to lightning_install.info.yml? This essentially forces all of Lightning to be 8.6+.